content/tools/plugins/JenkinsPlugin.md

```diff
@@ -8,42 +8,41 @@ tags: "tools Jenkins SDK Integration Java"
+#### Test the connection
+
+When you add a Contrast profile, use the **Test TeamServer Connection** button to test your connection and make sure that all the fields are accurate. Contrast prompts you if the test is successful, or gives an error message if it fails.
 
 ### Threshold conditions in a post-build action
 
-* Select a profile from the dropdown.
-* Add an application name. The Jenkins job name is used by default for this field. The field is **required**.
-* Add a count. The count is exclusive, so if you set a count for five, it fails on six or more vulnerabilities. This field is **required**.
-* Add a severity (Note, Low, Medium, High, or Critical). The plugin sets a filter in the API call for all vulnerabilities greater than or equal to this field. This field is recommended to reduce your results, but not required. 
-* Add a vulnerability type (rule name). If you specify a single rule for which to filter, the plugin checks for the number of vulnerabilities with the rule type and compares it to the count. This field is recommended to reduce your results, but not required. 
-* Add vulnerability statuses. It's not required, but can be helpful if you want to exclude vulnerabilities with certain statuses - for example, "Not a Problem" - from the results. If no statuses are selected, the plugin will not filter vulnerabilities by statuses.
+Once a connection is made, complete the following fields for **Post-Build Actions**. 
+
+* Select a **Profile** from the dropdown.
+* Add an application name. The Jenkins job name is used by default for this field. This field is **required**.
+* Add a **Count**. The count is exclusive; if you set a count for "5", it fails on six or more vulnerabilities. This field is **required**.
+* Choose a **Severity** from the options in the dropdown menu (Note, Low, Medium, High or Critical). The plugin sets a filter in the API call for all vulnerabilities greater than or equal to this field. This field is recommended to reduce your results, but not required. 
+* Choose a **Vulnerability Type** (rule name) from the dropdown menu. If you specify a single rule for which to filter, the plugin checks for the number of vulnerabilities with the rule type and compares it to the count. This field is recommended to reduce your results, but not required. 
+* Add vulnerability statuses. Statues aren't required, but can be helpful if you want to exclude vulnerabilities with certain statuses - for example, "Not a Problem" - from the results. If you don't select any statuses, the plugin won't filter vulnerabilities by statuses.
+
+<a href="assets/images/Jenkins_threshold_condition.png" rel="lightbox" title="Threshold condition configuration"><img class="thumbnail" src="assets/images/Jenkins_threshold_condition.png"/></a>
```

Comment:
Is this an up-to-date image? I don't see the application name or vulnerability statuses fields mentioned in the text.

Reply:
@kaitlynmortimer I have just pushed the updated images